keps/sig-cluster-lifecycle/wgs/1815-componentconfig-objectmeta/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: Support ObjectMeta in ComponentConfig API's
+kep-number: 1815
+authors:
+  - "@phenixblue"
+owning-sig: sig-cluster-lifecycle
+participating-sigs:
+  - sig-cli
+status: provisional
+creation-date: 2020-05-27
+reviewers:
+  - "@mtaufen"
+approvers:
+  - "@mtaufen"
+see-also:
+  - NA
+replaces:
+  - NA
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: stable
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.20"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.20"
+  beta: "v1.21"
+  stable: "v1.22"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: MyFeature
+    components:
+      - kube-apiserver
+      - kube-controller-manager
+disable-supported: true
```

Comment:
Maybe `false` to be safe since if we add it to `beta` schemas we can't remove it later without a fair amount of work?